audit_manager.tf

```diff
@@ -0,0 +1,41 @@
+resource "aws_auditmanager_account_registration" "default" {
+  count = var.aws_auditmanager_config.enabled == true ? 1 : 0
+
+  delegated_admin_account = data.aws_caller_identity.audit.account_id
+  deregister_on_destroy   = true
+  kms_key                 = module.kms_key_audit.arn
+
+  depends_on = [
```

Comment:
is this needed?

Reply:
You mean the `depends_on`? It's needed because TF might fail if it tries to add the KMS key in audit manager before the KMS key policy is updated

Reply:
Yeah but that should be covered by this implicit dependency right?
kms_key                 = module.kms_key_audit.arn

Reply:
Good point, overlooked that, will update!